RecoMuon/MuonIdentification/src/CSCTimingExtractor.cc

```diff
@@ -256,7 +249,22 @@ CSCTimingExtractor::fillTiming(TimeMeasurementSequence &tmSequence, reco::TrackR
 
   tmSequence.totalWeightInvbeta=totalWeightInvbeta;
   tmSequence.totalWeightTimeVtx=totalWeightTimeVtx;
+}
 
+
+// ------------ method called to produce the data  ------------
+void
+CSCTimingExtractor::fillTiming(TimeMeasurementSequence &tmSequence, reco::TrackRef muonTrack, 
+                               const edm::Event& iEvent, const edm::EventSetup& iSetup)
+{
+
+  if (debug) 
+    std::cout << " *** CSC Timimng Extractor ***" << std::endl;
```

Comment:
In the DTTimingExtractor.cc you decided to let the debug cout inside the called fillTiming(...)
(Just noticing the difference, in case you want to make unifom: I have no preference for it)

Reply:
That's not my code. I just moved it around to be able to use segments as input.